user/enterprise/operations-manual.md

```diff
@@ -0,0 +1,130 @@
+- `Worker machine`: The worker machine(s) run your builds.
+
+## Backup your encryption key
+
+This key is a very important building block in Travis CI Enterprise. It is used to encrypt certain parts of your production database. _Without this key the information in the database is not recoverable._ To make sure that the database remains accessible we ask you to make a backup of this key by following the steps below:
+
+- open a ssh connection to the platform machine
+- run `travis bash`. This will open a bash session with `root` privileges into the Travis container.
+- Then run `cat /usr/local/travis/etc/travis/config/travis.yml | grep -A1 encryption:`. Create a backup of value getting printed on the screen by either  writing it down on a piece of paper or storing it on a different computer.
+
+## Create a backup of the data directories
+
+The data directories are located on the platform machine and get mounted into the Travis container. In there you'll find files from RabbitMQ, Postgres, Slanger and Postgresql.
+
+These files are located at `/var/travis`. Please run `sudo tar -czvf travis-enterprise-data-backup.tar.gz /var/travis` to create compressed archive from this folder. After this has finished, copy this file off the machine to a secure location.
+
+## Pull a Support bundle
+
+Please head over to https://<your domain>:8800/support and create a support bundle. If you're writing us, please include that file. It'll include all important log files which will help us to diagnose the problem.
+
+Also please include the worker log files as well. They're located on your worker machine(s) at `/var/log/upstart/travis-worker.log`. If you run multiple worker machines, please grab this file from all machines.
+
+## What exactly is not running?
+
+What unusual behaviour are you seeing right now?
+
+- A build does not get picked up?
+- A build does not get enqueued after a `git push` or a freshly created Pull Request?
+- ...
+
+When you're writing to us, please include this information. We're also interested in recent changes in your IT landscape. For instance: Did your GitHub Enterprise Installation receive an update recently?
+
+## Firewall
+
+Are all machines online? Can they talk to each other? Please check this beforehand. If you're using a firewall, please make sure that these ports are open and the machines accept traffic on these:
+
+__Worker:__
+
+- Port 22 for SSH access
+
+__Platform machine__:
+
+- Port 22 for SSH access
+- Port 443 for HTTPS traffic
+- Port 80 for HTTP traffic
+- Port 4567 & 5672 for RabbitMQ traffic
+- Port 8800 for management dashboard https traffic
+
+## Hardware
+
+Our platform machine has these hardware requirements:
+
+- Minimum 16GB of RAM
+- 8 CPU cores
+- Min. 40GB of available disk space
+
+When you run `top` on the platform machine, please check if enough RAM is available and that the system load is not too high.
```

Comment:
Enough ram and load not too high is not specific. People might not know systems well enough to answer this

Reply:
☝️ @plaindocs with the really good points
Should we ask folks to check the pre-flight checks and let us know what they get? There are some false alarms in there on RAM, if I recall, but it might help with that and other stuff too